snapshot/git/gitdb/db_test.go

```diff
@@ -119,26 +117,61 @@ func TestStream(t *testing.T) {
 
 	streamID := snap.ID("stream-gc-sm-" + upstreamCommit1ID)
 
-	co, err := fixture.db.Checkout(streamID)
+	co, err := fixture.simpleDB.Checkout(streamID)
 	if err != nil {
 		t.Fatal(err)
 	}
 
+	defer fixture.simpleDB.ReleaseCheckout(co)
+
 	if err := assertFileContents(co, "file.txt", "upstream_first"); err != nil {
 		t.Fatal(err)
 	}
 
 }
 
+func TestTags(t *testing.T) {
+	// Create a commit in external, then ingest into authorDB, then check it out in our regular db
+
+	externalCommitID, err := commitText(fixture.tags, "tags_first")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	id, err := fixture.authorDB.IngestGitCommit(fixture.tags, externalCommitID)
```

Comment:
The public (exported) method still returns an ID; it's ingestGitCommit that now returns a snapshot.